app/forms/withdrawal/date_form.rb

```diff
@@ -19,8 +19,7 @@ def fields
     end
 
     def save!
-      assign_attributes_to_trainee
-      trainee.save
+      trainee.trainee_withdrawals.last.update!(date: withdraw_date)
```

Comment:
I think this will need to be made more robust - in general the actual "save" action in the final stage is not ideal. however I'd like to get this merged to the epic with a tidy up at the end once we have the journey fully complete with specs to match, should make refining a bit easier